java_collections_block/optional_tasks/src/main/java/Task1.java

```diff
@@ -0,0 +1,33 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class Task1 {
+
+//    Ввести строки из файла, записать в список. Вывести строки в файл в обратном порядке.
+
+    public static void task1() {
+        String inputFileName = "java_collections_block/optional_tasks/src/main/resources/Input_file_for_task1.txt";
+        String outputFileName = "java_collections_block/optional_tasks/src/main/resources/Output_file_for_task1.txt";
+        List<String> listOfFileLines = new ArrayList<>();
+
+        try {
+            BufferedReader br = Files.newBufferedReader(Paths.get(inputFileName));
+            //br returns as stream and convert it into a List
+            listOfFileLines = br.lines().collect(Collectors.toList());
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
In the current implementation you will just print stack trace and continue execution
I suggest throwing unchecked exception here

like replace this line by 
```
throw new IllegalStateException("Cannot read file", e)
```

In this case your program will be aborted in case of weird things with file

Reply:
done